src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const obj = sourceString
+    .split(';')
+    .filter((style) => style.includes(':'))
+    .reduce((prev, style) => {
+      const [key, value] = style.split(':');
+
+      if (key && value) {
+        prev[key.trim()] = value.trim();
```

Comment:
Trimming the key and value is a good practice to remove any leading or trailing whitespace. This ensures that the resulting object has clean keys and values.